modules/fx/arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/MonoK.kt

```diff
@@ -85,11 +85,11 @@ data class MonoK<out A>(val mono: Mono<out A>) : MonoKOf<A> {
    */
   fun <B> bracketCase(use: (A) -> MonoKOf<B>, release: (A, ExitCase<Throwable>) -> MonoKOf<Unit>): MonoK<B> =
     MonoK(Mono.create<B> { sink ->
-      val isCanceled = AtomicBoolean(false)
-      sink.onCancel { isCanceled.set(true) }
+      val isCanceled = Atomic(false)
```

Comment:
Knowing JBs I'm surprised there isn't a specialised boolean/int.

Reply:
There is, I'll create wrappers and replace the refs ;)